blackbox/docs/sql/information_schema.txt

```diff
@@ -150,6 +150,75 @@ You can even query this tables' own columns (attention: this might lead to infin
   position.
 
 
+The following columns according to the SQL99 standard are available:
```

Comment:
Not all these columns are in the SQL99 standard - I think rewording this to be "The specification of columns, which is compliant with SQL99" if we really wanted to mention it would work better